client/controller/index.node.js

```diff
@@ -82,6 +82,8 @@ export const redirectLoggedOut = () => {};
 export const redirectLoggedOutToSignup = () => {};
 export const redirectWithoutLocaleParamIfLoggedIn = () => {};
 // eslint-disable-next-line no-unused-vars
+export const redirectIfCurrentUserCannot = ( capability ) => {};
```

Comment:
Is this necessary/appropriate?

Reply:
@tyxla ? :-)

Reply:
@Automattic/team-calypso-frameworks

Reply:
Looks good. We could likely ditch the unused argument (and the ESLint ignore) though, I don't think there's a good reason to keep it.